packages/markdown/src/Markdown.tsx

```diff
@@ -107,30 +112,32 @@ function Markdown({
                   withModal={withModal}
                 />
               ),
-              a: ({
-                href,
-                children,
-              }: {
-                href: string
-                children: React.ReactNode
-              }) => {
-                const isExcel = href.includes('.xls')
-                const isPDF = href.includes('.pdf')
-                return (
-                  <a
-                    className={twMerge(
-                      'px-4 py-3 border rounded sm:hover:bg-slate-200 flex flex-row gap-3 text-sm my-1'
-                    )}
-                    href={href}
-                  >
-                    <div>
-                      {isExcel && <FontAwesomeIcon icon={faFileExcel} />}
-                      {isPDF && <FontAwesomeIcon icon={faFilePdf} />}
-                    </div>
-                    <div>{children}</div>
-                  </a>
-                )
-              },
+              a: withLinkButtons
```

Comment:
we do not want to parse links to buttons in all use cases (e.g., in the privacy notes), so this is a new prop to disable this functionality